### PR TITLE
Added travis .yml file

### PR DESCRIPTION
**Pull request by amadeu01**
I just added `travis.yml`.
It simply runs the tests. 
One step further would be to add coverage. 

`It will fail due to some mistyped imports`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+sudo: required
+
+language: ruby
```
Comment (cuducos) [inline .travis.yml]: Why Ruby? I know everything is going to run inside the Docker container… so either we skip `language` (if Travis allow us to do so) or we declare Python, which is what we really use. Does that make sense?

Comment (amadeu01) [reply .travis.yml]: I really thought about replacing `ruby`. I just was used to it. But, nevermind, I'll change it.

**Comment by cuducos** [comment]
Just added the repo on Code Climate (#28). What about adding badges in the `README.md` and coverage in this PR too?

**Comment by amadeu01** [comment]
For sure!

One thing the link for the badge is on the settings panel of the Code Climate. So, I do not have access to it. You can update this PR with the badge. 

I'll be adding the badge for travis and coverage.

**Comment by cuducos** [comment]
> One thing the link for the badge is on the settings panel of the Code Climate

It's is, but it's awful that Code Climate offers only its customized badges, that doesn't match others badges. Aesthetically it is terrible. What about using a single provider for all badges such as [shields.io](http://shields.io/) so all badges match?

**Comment by amadeu01** [comment]
Sorry, I forgot to answer you. I'll do it. I had never used shields.io, but that will be a great first time hehe.
I just cannot do it today. But, I'll do as soon as I get some free time.

**Comment by amadeu01** [comment]
@cuducos I added the badges from shield.io

**Comment by amadeu01** [comment]
@cuducos I guess you are still fixing the docker, so I'll wait until you fix it, then I will also update the travis to also generate the coverage report. 😄

**Comment by amadeu01** [comment]
@cuducos I did the coverage report and sent it to codecov on the fork [amadeu01/perfil/tree/add-coverage](https://github.com/amadeu01/perfil/tree/add-coverage)

But, I also changed the pytest.ini and created `test_settings.py`. So, I do not know it that is acceptable to you. So, if you had the time to check it out. If you think that is fine, I can make other PR.

**Comment by cuducos** [comment]
> I did the coverage report and sent it to codecov

Do we have a badge for that too?

> I also changed the `pytest.ini` and created `test_settings.py`

I cannot see this reflected in this PR. Can you double-check if you've added the files?

**Comment by amadeu01** [comment]
@cuducos Sorry for the misunderstanding. I meant I can do other PR. I'll make other PR, if you do not like it, just close it hahaha

**Comment by cuducos** [comment]
>  I meant I can do other PR

Why another PR if this PR here is dealing with the CI setup? I do think these proposals could be added here ; )

**Pull request merged**